models/tmp/stg_iterable__user_unsubscribed_channel_tmp.sql

```diff
@@ -0,0 +1,12 @@
+
+{% if does_table_exist('user_unsubscribed_channel') %}
+
+select *
+from {{ var('user_unsubscribed_channel') }}
+
+{% else %}
+
+select *
+from {{ var('user_unsubscribed_channel_history') }}
+
+{% endif %}
```

Comment:
Same suggestion from the unsub message type tmp model about making the code a bit more concise.

```suggestion
select *
from {{ var('user_unsubscribed_channel') if does_table_exist('user_unsubscribed_channel') else var('user_unsubscribed_channel_history') }}
```

Reply:
updated